components/common/ui/IconImage.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const IconImage = ({ className, name }) => (
+  <>
+    {name === 'molamola' && (
```

Comment:
it is better for the bundle size if 1 icon = 1 component, so replace with `MolamolaIcon`